app/javascript/packs/Main.jsx

```diff
@@ -26,6 +27,10 @@ export default class Main extends React.Component {
           <Route exact path='/users/sign_out' render={() => (
              <LogOut />
           )} />
+
+          <Route path='/users/*' render={() => (
```

Comment:
any reason you're targeting all paths after /users/*?

Reply:
yea this is definitely too broad of a match. Trying to match only as far as the users id for example '/users/:id'